apstools/utils/device_info.py

```diff
@@ -31,10 +33,10 @@
 NOT_CONNECTED_VALUE = "-n/c-"
 
 
-def _all_signals(base):
+def _all_signals(base: Union[Signal, Device]) -> list[Signal]:
```

Comment:
Looking at [`OphydObject`](https://github.com/bluesky/ophyd/blob/496fae33a8181307ab199d18fc4e32b14248b6e3/ophyd/ophydobj.py#L115) as the ultimate base class.  Its the base class for [`Device`](https://github.com/bluesky/ophyd/blob/496fae33a8181307ab199d18fc4e32b14248b6e3/ophyd/device.py#L784C32-L784C43) and [`Signal`](https://github.com/bluesky/ophyd/blob/496fae33a8181307ab199d18fc4e32b14248b6e3/ophyd/signal.py#L79):

```suggestion
def _all_signals(base: Union[OphydObject]) -> list[Signal]:
```